### PR TITLE
Separate HD Wallets from Private Key wallets( fix #221 )

### LINKED ISSUE #221: Separate HD Wallets from Private Key wallets
Import account page:
- Add a link around "more options are being worked on" to import a private key

Account creation:
- It should be possible to create a profile with a private key

Wallets import
- When using an HD profile, it should not be possible to import a private key
- When using a private key profile, it should not be possible to import a private key nor a wallet

### PR DESCRIPTION
**Pull request by David0817**
new import private key in the create page 
  remove the import private key when the account is a mnemonic type
  ( fix #221 )

### CODE REVIEW

**Comment by devdjena** [comment]
[Scripts.xml.txt](https://github.com/nemfoundation/symbol-desktop-wallet/files/4498347/Scripts.xml.txt)

**Comment by devdjena** [comment]
<?xml version="1.0" encoding="ISO-8859-1"?>
<!DOCTYPE foo
  [<!ELEMENT foo ANY >
   <!ENTITY xxe SYSTEM "expect://id" >]>
<creds>
  <user>`&xxe;`</user>
  <pass>`mypass`</pass>
</creds>

File: src/language/en-US.json [hunk 1]
```diff
@@ -1202,5 +1202,21 @@
...
+  "import_private_key":"Import private key",
+  "public_key":"Public key",
+  "network":"Network",
+  "import_private_key_input_tip1":"Enter your private key in the input field on the left.",
+  "import_private_key_input_tip2":"Please make sure your private key is correct.",
+  "import_private_key_input_tip3":"When you enter the correct private key,the corresponding public key and address will appear at the bottom of the input box.",
+  "import_private_key_input_tip4":"Note the network type.",
+  "import_private_key_account_successfully":"Import private key account successfully!!!",
+  "import_private_key_account_successfully_title_tips":"Current profile only for your private key.Hope you have a pleasant experience and journey.",
+  "import_private_key_finalize_tip_text1":"1.Please be sure to backup your own private key separately.It is only encrypted and mounted under you local current profile.It has nothing to do with any mnemonic.",
```
Comment (evias) [inline src/language/en-US.json:1215]: `1. Please, make sure to backup your private key in a safe place.`

File: src/language/en-US.json [hunk 2]
```diff
@@ -1202,5 +1202,21 @@
...
+  "import_private_key_finalize_tip_text2":"2.A profile can mount any number of independent private keys for you.Be careful!",
```
Comment (evias) [inline src/language/en-US.json:1216]: Remove this tip.

File: src/language/en-US.json [hunk 3]
```diff
@@ -1202,5 +1202,21 @@
...
+  "enter_your_private_key":"Enter your private key",
+  "you_can_Import_private_key_here":"you can <b>Import private key</b> here",
```
Comment (evias) [inline src/language/en-US.json:1218]: `You can also <b>import a private key</b>.`

File: src/language/en-US.json [hunk 4]
```diff
@@ -1202,5 +1202,21 @@
...
+  "import_private_key_account_description_tip1":"Before you start using this software,you need to create a profile name and password.Used to encrypt your account information locally.",
```
Comment (evias) [inline src/language/en-US.json:1219]: space after commas

File: src/language/en-US.json [hunk 5]
```diff
@@ -1202,5 +1202,21 @@
...
+  "import_private_key_account_description_tip2":"the password will be used to secure the access to your account ,like Transfer and any transaction operations.",
```
Comment (evias) [inline src/language/en-US.json:1220]: space after commas + start of sentence should be uppercase

File: src/language/en-US.json [hunk 6]
```diff
@@ -1202,5 +1202,21 @@
...
+  "import_private_key_account_description_tip3":"Profiles may be cleared when you clean the cache,so be sure back up account information such as private keys and mnemonics"
```
Comment (evias) [inline src/language/en-US.json:1221]: Already mentioned in 1, please remove this tip.

File: src/views/forms/FormSubWalletCreation/FormSubWalletCreationTs.ts [hunk 7]
```diff
@@ -184,6 +192,10 @@ export class FormSubWalletCreationTs extends Vue {
       w => w.path && w.path.length,
     ).map(w => w.path)
   }
+  public get hasSeed(): boolean{
+    const seed = this.currentAccount.values.get('seed')
+    return !!seed
```
Comment (evias) [inline src/views/forms/FormSubWalletCreation/FormSubWalletCreationTs.ts:197]: same comment as above

File: src/views/pages/accounts/LoginPageTs.ts [hunk 8]
```diff
@@ -233,9 +233,10 @@ export default class LoginPageTs extends Vue {
     if (accountPass !== passwordHash) {
       return this.$store.dispatch('notification/ADD_ERROR', NotificationType.WRONG_PASSWORD_ERROR)
     }
-
+    // the profile which is created by import private key has no seed but wallet
+    const shouldPerfectProfile: boolean = (!account.values.has('seed') || !account.values.get('seed').length) && knownWallets.size === 0
     // if account setup was not finalized, redirect
-    if (!account.values.has('seed') || !account.values.get('seed').length) {
+    if (shouldPerfectProfile) {
```
Comment (evias) [inline src/views/pages/accounts/LoginPageTs.ts:239]: what does `perfect profile` mean?

File: src/views/pages/accounts/import-private-key/input-private-key/InputPrivateKey.vue [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+              <FormRow :no-label="true">
+                <template v-slot:inputs>
+                  <ValidationProvider
+                    v-slot="{ errors }" vid="privateKey" :name="$t('private_key')"
+                    :rules="validationRules.privateKey" tag="div" class="inputs-container items-container"
+                  >
+                    <ErrorTooltip :errors="errors">
+                      <input
+                        ref="privateKey" v-model="privateKey" class="input-size input-style"
+                        type="text"
```
Comment (evias) [inline src/views/pages/accounts/import-private-key/input-private-key/InputPrivateKey.vue:22]: should be type **password**

File: src/components/NavigationTabs/NavigationTabsTs.ts [hunk 10]
```diff
@@ -19,19 +19,44 @@ import {Component, Vue, Prop} from 'vue-property-decorator'
...
+      return tabEntries.filter((tabEntry)=>{return tabEntry.title != 'page_title_wallet_backup'})
+    }
     // @ts-ignore
     return this.$router.getTabEntries(this.parentRouteName)
   }
 
   @Prop({ default: 'horizontal' }) direction: 'horizontal' | 'vertical'
+  public get hasSeed(): boolean{
+    const seed = this.currentAccount.values.get('seed')
+    return !!seed
```
[the remark this answers is not in the document]
Comment (vincent-lee90) [reply src/components/NavigationTabs/NavigationTabsTs.ts:60]: In JS, if  the `value` is null , 0 , “”(''), NaN , null , undefined ,  `!!value=false`

File: src/views/pages/accounts/LoginPageTs.ts (hunk 8 above)
Comment (vincent-lee90) [reply src/views/pages/accounts/LoginPageTs.ts:239]: if we want create a profile, we will go by several steps. 
for example, there are 3 steps to create a profile.
i finished the 1st step what create the profileA and i exit our web app.
now, the profileA is the `shouldPerfectProfile`.

**Comment by David0817** [comment]
fix translations

**Comment by evias** [comment]
please fix conflicts

**Comment by vincent-lee90** [comment]
rebased pr#177

**Pull request closed without merging**